alternator/serialization.cc

```diff
@@ -59,7 +59,9 @@ struct from_json_visitor {
         bo.write(t.from_string(rjson::to_string_view(v)));
     }
     void operator()(const bytes_type_impl& t) const {
-        bo.write(rjson::base64_decode(v));
+        // FIXME: it's difficult at this point to get information if value was provided
+        // in request or comes from the storage, for now we assume it's user's fault.
+        bo.write(*unwrap_bytes(v, true));
```

Comment:
It's scary that unwrap_bytes() looks like it can return an unset optional even if the second parameter is true. But we're safe because it can't. Maybe this needs to be documented in a comment on unwrap_bytes() or here in the call.

Reply:
will add to unwrap_bytes docs, it's kind of mentioned but indirectly, should be more clear